networkmanager/ipamalloc.go

```diff
@@ -62,10 +67,8 @@ func (ipam *ipSubnetwork) tryToGetExistIPNetFromPool(spID string) (allocIPNet *n
 }
 
 func (ipam *ipSubnetwork) requestIPNetPool(spID string) (allocIPNet *net.IPNet, usedIPNet bool, err error) {
-	allocIPNet, usedIPNet = ipam.tryToGetExistIPNetFromPool(spID)
```

Comment:
You removed some functionality. Is it ok?

Reply:
yes